src/sentry/rules/conditions/event_attribute.py

```diff
@@ -78,7 +78,7 @@ class EventAttributeCondition(EventCondition):
     # TODO(dcramer): add support for stacktrace.vars.[name]
 
     form_cls = EventAttributeForm
-    label = "The event's {attribute} value {match} {value}"
+    label = "The event’s attribute matches..."
```

Comment:
I have a feeling this is used to set the layout of the dropdowns one you select a specific item. Could you fire up your devenv and confirm whether that's the case.

Reply:
oh yea, I think it is